tests/test_plugin.py

```diff
@@ -199,8 +200,11 @@ def test_screenshot(simple_page, browser):
     assert False
     """.format(simple_page_content), "-vl", "-r w")
 
-    assert testdir.tmpdir.join('test_browser_screenshot_normal', 'test_screenshot-browser.png')
-    content = testdir.tmpdir.join('test_browser_screenshot_normal', 'test_screenshot-browser.html').read()
+    dir_content = os.listdir('test_browser_screenshot_normal')
```

Comment:
a bit strange that tests are changed with the replacement of the screenshot procedure
those tests should pass without any change to keep the compatibility in tact